### ISSUE TITLE
Add support for completions for custom resource definitions

### ISSUE THREAD

**Issue by evanlucas** [evanlucas/fish-kubectl-completions#6]
Not sure if this is the most efficient way, but the following works to get custom resource definitions.

```
kubectl get crd -o yaml | egrep '      plural|      singular' | awk '{print $2}' | sort -u
```

Do CRDs have a description? If so, it would be cool to include those in the resources.

**Comment by evanlucas** [comment]
`kubectl get crd -o jsonpath='{.items[*].spec.names[\'plural\',\'singular\']}'` would probably be a better way to include those

**Comment by evanlucas** [comment]
Initial implementation was landed in e873cd9acc9cfd4dceaec9c43beb31b022f01236
